tests_e2e/pipeline/pipeline.yml

```diff
@@ -44,23 +50,6 @@ parameters:
     - failed
     - no
 
-variables:
```

Comment:
These variables are used only by the "Execute tests" tasks, so I removed them and referenced the parameters directly in that task.